### PR TITLE
Add ecemped column list for transliteration

### PR DESCRIPTION
**Pull request by afazdeen**
Add an exempted column list to handle the columns that do not need to be transliterated in demographic details form

### CODE REVIEW

File: registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java [hunk 1]
```diff
@@ -1757,6 +1757,16 @@ private void setTextFieldListener(TextField textField) {
 					LOGGER.debug(loggerClassName, APPLICATION_NAME, RegistrationConstants.APPLICATION_ID,
 							"Setting secondary langauge text");
 
+					// excempted column list
+					String[] exemptedTransliteration = new String[]{"email", "nationalIdentityNumber", "phoneNumber"};
+					List<String> exemptList = new ArrayList<>(Arrays.asList(exemptedTransliteration));
```
Comment (ghost) [inline registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java]: Arrays.asList(exemptedTransliteration) itself returns List<String> so this statement can be changed to 

List<String> exemptList = Arrays.asList(exemptedTransliteration);

File: registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java [hunk 2]
```diff
@@ -1757,6 +1757,16 @@ private void setTextFieldListener(TextField textField) {
 					LOGGER.debug(loggerClassName, APPLICATION_NAME, RegistrationConstants.APPLICATION_ID,
 							"Setting secondary langauge text");
 
+					// excempted column list
+					String[] exemptedTransliteration = new String[]{"email", "nationalIdentityNumber", "phoneNumber"};
```
Comment (ghost) [inline registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java]: Why not take this from properties ?

File: registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java (hunk 1 above)
Comment (afazdeen) [reply registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java]: Updated the PR, retrieved from properties, also fixed a issue in ClientSettingSyncHelper, where master sync fails if there are dynamic columns with same name for different languages

**Comment by sowmya695** [comment]
Closing the PR as confirm by  @ase-101

**Pull request closed without merging**